### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by shaardie**
Added README to this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# gitlab-jira-migration
+
+If you are using [JIRA](https://de.atlassian.com/software/jira) and [Gitlab](https://about.gitlab.com/) and you would like to change the url, user or password of your JIRA instance, you can not simply change the template and all the JIRA integration of all projects are changes (see [issue](https://gitlab.com/gitlab-org/gitlab-ce/issues/25541)).
+So you have to change every single project which is pretty anying. 
+
+Or you can use this script.
+
+## Dependencies
+
+You need [Requests](http://docs.python-requests.org/en/master/) for HTTP calls.
```
Comment (tibroc) [inline README.md:12]: Python Version? There is still a lot of Python 2 out there

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# gitlab-jira-migration
+
+If you are using [JIRA](https://de.atlassian.com/software/jira) and [Gitlab](https://about.gitlab.com/) and you would like to change the url, user or password of your JIRA instance, you can not simply change the template and all the JIRA integration of all projects are changes (see [issue](https://gitlab.com/gitlab-org/gitlab-ce/issues/25541)).
+So you have to change every single project which is pretty anying.
```
Comment (tibroc) [inline README.md]: typo: annoying

File: README.md (hunk 1 above)
Comment (shaardie) [reply README.md:12]: Should work both, but agree that this should be clarified.

**Review by tibroc — APPROVED** [review]
Looks good to me.

**Pull request merged**